frontend/src/services/services.js

```diff
@@ -84,6 +84,20 @@ export default {
     return x.toPrecision(3);
   },
 
+  prettifyJSON(jsonObj) {
+    let prettifiedString = "";
+    for (const key in jsonObj) {
+      if (jsonObj.hasOwnProperty(key)) {
+        prettifiedString += `${key} : ${jsonObj[key]}\n`;
+      }
+    }
+    // Remove trailing newline character if needed
+    if (prettifiedString.endsWith("\n")) {
+      prettifiedString = prettifiedString.slice(0, -1);
+    }
+    return prettifiedString;
+  },
```

Comment:
Does this function support nested json ?
The issue ticket dict : 
![image](https://github.com/debiai/DebiAI/assets/59999324/6fc1ed8a-be70-4dbb-b2e7-54ace811069f)

Reply:
Does debiAI create_model function support nested json ?

Reply:
It does not. I will implement a recursive version of the function to do so.

Reply:
The metadata parameter of the create_model() function should, yes